proposals/acp-specification/index.html

```diff
@@ -0,0 +1,2561 @@
+            request attributes, it should advertise it by responding to HTTP
+            OPTIONS requests over ACP access control resources including three
+            link headers with rel value of
+            <code>http://www.w3.org/ns/solid/acp#attribute</code> and
+            respectively targets of
+            <code>http://www.w3.org/ns/solid/acp#agent</code>,
+            <code>http://www.w3.org/ns/solid/acp#client</code> and
+            <code>http://www.w3.org/ns/solid/acp#issuer</code> in order to make
+            it explicit which set of request attributes are understood and
+            relevant when editing ACP matchers.
+          </p>
+
+          <section
+            datatype="rdf:HTML"
+            typeof="bibo:Chapter"
+            resource=":options-discovery"
+            property="bibo:hasPart"
+          ></section>
+        </section>
+        <h2 class="heading settled" data-level="6" id="solid-acp">
+          <span class="secno">6. </span><span class="content">Solid ACP</span
+          ><a class="self-link" href="#solid-acp"></a>
+        </h2>
+        <p>
+          This section introduces how ACP may be used in the Solid ecosystem.
+        </p>
+
+        <section
+          datatype="rdf:HTML"
+          typeof="bibo:Chapter"
+          resource=":solid-acp"
+          property="bibo:hasPart"
+        >
+          <h3 class="heading settled" data-level="6.1" id="resource-management">
+            <span class="secno">6.1. </span
+            ><span class="content">Resource management</span
+            ><a class="self-link" href="#resource-management"></a>
+          </h3>
+          <p>Access control resources MUST be server managed.</p>
+          <p>
+            There is a one to one relationship between an access control
+            resource and the resource it mandates access permissions over.
+            Therefore, when a resource is created or deleted, the access control
+            resource corresponding to it SHOULD be created or deleted
+            accordingly.
+          </p>
+          <p>
+            Access controls, policies and matchers can all be kept in separate
+            resources that have their own access control resource mandating
+            access permissions over them.
+          </p>
+          <p>
+            As long as an access control, a policy or a matcher is referenced by
+            an access control resource, it SHOULD not be deleted. However, if
+            such consistency cannot be enforced and an access control, a policy
+            or a matcher ceased to be accessible, then access permissions
+            resolution should fail to its default behaviour of only granting
+            read and write access to the failing access control resource to its
+            owner(s); read and write access should not be implicitly granted to
+            a resource whose access control resolution fails, even to owners of
+            that resource.
```

Comment:
Does it mean that owner still can amend access control so that the resolution stop failing?

Reply:
If he is the owner of the resource where the access control is defined, eventually yes.
In any case an owner can always edit out problematic access controls (as well as policies and matchers) and replace them with valid ones, so ultimately yes.
An owner can always fixup and modify access to a resource as they see fit.